common/data_refinery_common/migrations/0043_processorjob_abort.py

```diff
@@ -0,0 +1,60 @@
+from django.db import migrations, models
+
+
+def batch_update(queryset, batch=1000, **changes):
+    """ Update per batch """
+    is_done = False
+    current = 0
+
+    while not is_done:
+        current = current + 1
+        start = (current - 1) * batch
+        end = start + batch
+        pks = queryset.values_list('pk', flat=True)[start:end][::1]
+        if pks:
+            queryset.model.objects.filter(pk__in=pks).update(**changes)
+
+        is_done = len(pks) < batch
+
+
+def set_default_abort_value(apps, schema_editor):
+    ProcessorJob = apps.get_model('data_refinery_common', 'ProcessorJob')
+    queryset = ProcessorJob.objects.all()
+    batch_update(queryset, abort=False)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_refinery_common', '0042_remove_experiment_organism_names'),
```

Comment:
This will need to be updated because we've had migrations go out. Sorry about taking so long on this :(